src/sentry/templatetags/sentry_helpers.py

```diff
@@ -65,10 +65,58 @@ def coerce(value):
     return coerce(x) * coerce(y)
 
 
-@register.simple_tag
-def absolute_uri(path='', *args):
-    from sentry.utils.http import absolute_uri
-    return absolute_uri(path.format(*args))
+class AbsoluteUriNode(template.Node):
+    def __init__(self, args, target_var):
+        self.args = args
+        self.target_var = target_var
+
+    def render(self, context):
+        from sentry.utils.http import absolute_uri
+
+        # Attempt to resolve all arguments into actual variables.
+        # This converts a value such as `"foo"` into the string `foo`
+        # and will look up a value such as `foo` from the context as
+        # the variable `foo`. If the variable does not exist, silently
+        # resolve as an empty string, which matches the behavior
+        # `SimpleTagNode`
+        args = []
+        for arg in self.args:
+            try:
+                arg = template.Variable(arg).resolve(context)
+            except template.VariableDoesNotExist:
+                arg = ''
```

Comment:
Might be worth throwing, at least if `DEBUG` is enabled? This seems like an easy mistake to make and hard to catch since these usually end up as tag attributes and not text nodes.

Reply:
I'd argue that this falls in line with just how Django templates work. You can do `{{ doesnotexist }}` and it silently fails and does nothing. It's an entirely different `setting` to enable to change this behavior. And considering this is how it was before, I'm inclined to leave it as-is.